src/footer.js

```diff
@@ -0,0 +1,31 @@
+class Footer extends HTMLElement {
+    constructor() {
+      super();
+    }
+  
+    connectedCallback() {
+      this.innerHTML = `
+        <footer>
+          <ul class="side-map">
+            <li><a class="a-footer" href="about.html">About</a></li>
```

Comment:
Myślę że tą mapę strony możemy wywalić. W przyszłości raczej nie będziemy robić tych podstron: About, Terms of use, Contact

Reply:
Mapa strony wywalona.